doc/getting_started/installation_win.rst

```diff
@@ -27,28 +27,65 @@ The simplest way to install a development environment on Windows is to use
 MSYS2, a modern UNIX environment for Windows. Follow the steps below to set it
 up:
 
-1. Download and install :program:`MSYS2`. Download the appropriate (32 or
+#. Download and install :program:`MSYS2`. Download the appropriate (32 or
    64-bit) MSYS2 installer from the `MSYS2 website`_ and execute it. On the
    final installation screen, check the "Run MSYS2 now." box to start up an
    MSYS2 shell when installation is complete.  Follow the rest of the
    installation instructions on the MSYS2 website to update the package
    database and core system packages.  You may be advised to "terminate MSYS2
    without returning to shell and check for updates again".  If so, simply
-   close the ``MSYS2 MSYS`` desktop app and run it again to complete the update.)
+   close the ``MSYS2 MSYS Shell`` desktop app and run it again to complete the update.)
 
-#. Launch the ``MSYS2 MSYS`` desktop app from your start menu (if it's not still open).
+#. Launch the ``MSYS2 MSYS Shell`` desktop app from your start menu (if it's not still open).
+
+     .. note::
+        There are multiple ``export`` statements in this tutorial. In order for
```

Comment:
@dbkinder the problem is that some of them are optional depending on the user's needs (http proxy, DTC). If I consolidate them in a single place it might lead people to copy-paste them all